packages/next/server/hot-reloader.js

```diff
@@ -161,7 +161,7 @@ export default class HotReloader {
   }
 
   async clean () {
-    return del(join(this.dir, this.config.distDir), { force: true })
+    return promisify(rimraf)(join(this.dir, this.config.distDir), { force: true })
```

Comment:
This'll promisify it on every clean.  Want to hoist it?

Reply:
Agreed it should be initialized right after the imports.